linode/manager
# feat: [M3-6944]: Add Firewalls table to `Linode Details > Network` tab

## Description 📝
Add Firewalls table to `Linode Details > Network` tab.

## Major Changes 🔄
- "Firewalls" table added to the `Linode Details > Network` tab
- `LinodeNetworking.tsx` renamed to `LinodeIPAddresses.tsx` since it was specifically the IP Addresses section
- `LinodeNetwork.tsx` created as parent component of `LinodeNetworkingSummaryPanel`, `LinodeFirewalls`, and `LinodeIPAddresses`, named to match the tab name for clarity
- `index.tsx` files deleted according to our convention

## To-Do:
- [x] Logic cleanup + general cleanup
- [x] Simplify styling/formatting 
- [x] Squash bugs

## Preview 📷
<details>
<summary>No firewalls assigned</summary>

![Screenshot 2023-08-08 at 1 57 21 PM](https://github.com/linode/manager/assets/114682940/7008ee9a-996e-4f9c-8f06-66d7f7bc3b5e)
</details>

<details>
<summary>Firewalls assigned</summary>

![Screenshot 2023-08-08 at 1 57 44 PM](https://github.com/linode/manager/assets/114682940/6e9f1411-89f9-43c6-921d-32462885f8b5)
</details>

## How to test 🧪
- Ensure there have been no adverse impacts to display or functionality of the IP Addresses table or Networking Summary Panel on `Linode Details > Network` tab, and that you can unassign a firewall from this view (and that the table updates accordingly)
- Ensure there have been no adverse impacts to the Firewall row of the Firewalls landing page, or to the "Remove" functionality in the `Firewall Details > Linodes` tab
